src/QA/QA.Web/Mappings/ReviewMapper.cs

```diff
@@ -27,13 +27,13 @@ public class ReviewMapper
         private readonly IGeocodeImageService _mapService;
         private readonly IMinimumWageProvider _wageProvider;
         private readonly Lazy<IList<string>> _qualifications;
-        private readonly IRuleTemplateRunner _ruleTemplateRunner;
+        private readonly IRuleTemplateMessageRunner _ruleTemplateRunner;
```

Comment:
IRuleMessageTemplateRunner may be, matches MessageTemplate naming convention used else where